public/modules/custom/grants_budget_components/js/budget_total.js

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line no-unused-vars
+((Drupal, drupalSettings) => {
+  Drupal.behaviors.grants_budget_total_fieldAccessData = {
+    attach: function attach() {
+      Object.values(drupalSettings.totalFields).forEach(totalField => {
+        const totalFieldName = totalField.totalFieldId;
+
+        let fieldsArray = [];
+
+        let fieldArray = totalField.fields;
+
+        fieldArray.forEach(fieldNameArray => {
+          const fieldNameRaw = fieldNameArray.fieldName;
+          const fieldName = fieldNameRaw.replaceAll('_', '-');
+          const columnNameRaw = fieldNameArray.columnName;
+          const columnName = columnNameRaw.toLowerCase();
+          fieldsArray.push('edit-' + fieldName + '-' + columnName);
+        })
+
+        function calculateTotal() {
+          let sum = 0;
+          fieldsArray.forEach(item => {
+            const elementItem = document.getElementById(item);
+            let myString = '';
+
+            myString = 0 + elementItem.value;
+            myString = myString * 100;
+            sum += parseInt(myString);
+          })
+
+          sum = sum / 100;
+          document.getElementById(totalFieldName).value = sum + '';
+        }
+
+        calculateTotal();
+
+        fieldsArray.forEach(field => {
+          let myEle = document.getElementById(field);
```

Comment:
Jos on konffinu päin metsää elementin, ja tätä ei oo olemassa, niin kaataa koko laskurin toiminannan, varmaan joku sanity check, että löytyykö elementti oikeesti?